front/app/containers/Admin/reporting/components/ReportBuilder/Editor/RenderNode/index.tsx

```diff
@@ -49,24 +49,24 @@ const RenderNode = ({ render }) => {
 
   const {
     isActive,
-    isDeletable,
     parentId,
+    isDeletable,
     actions: { selectNode },
     query: { node },
   } = useEditor((_, query) => {
     return {
-      isActive: id && query.getEvent('selected').contains(id),
-      parentId: id && query.node(id).ancestors()[0],
-      isDeletable: id && query.node(id).isDeletable(),
+      isActive: id ? query.getEvent('selected').contains(id) : undefined,
```

Comment:
Just curious, why was this necessary?

Reply:
Slightly cleaner way to do typeguards. In the old way, `isActive` is `boolean` or an empty string. In the new way, it's a `boolean` or `undefined. Changed this while I was debugging another issue but just kept it like this because I like it more